library/init/category/cont.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright (c) 2018 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sebastian Ullrich
+
+The continuation monad transformer.
+-/
+
+prelude
+import init.category.alternative init.category.combinators init.category.lift
+universes u v w
+
+structure cont_t (r : Type u) (m : Type u → Type v) (α : Type u) : Type (max u v) :=
```

Comment:
Is this working in progress?
We don't have a `lift` like other transformers. 
AFAICT, this transformer is not used anywhere.
If you want to keep it, we need to document, add examples to test it, etc.